### PR TITLE
Added description attribute to schema_template_bd and schema_template_anp_epg resources

### PR DESCRIPTION
**Pull request by sajagana**
Note:

Added description attribute to schema_template_bd and schema_template_anp_epg resources and updated resource examples and documents.

### CODE REVIEW

File: mso/resource_mso_schema_template_anp_epg.go [hunk 1]
```diff
@@ -735,7 +744,7 @@ func resourceMSOSchemaTemplateAnpEpgDelete(d *schema.ResourceData, m interface{}
 	bdRefMap["bdName"] = bdName
 
 	path := fmt.Sprintf("/templates/%s/anps/%s/epgs/%s", templateName, anpName, d.Id())
-	anpEpgStruct := models.NewTemplateAnpEpg("remove", path, Name, displayName, intraEpg, epgType, uSegEpg, intersiteMulticasteSource, preferredGroup, proxyArp, vrfRefMap, bdRefMap, cloudServiceEpgConfig)
+	anpEpgStruct := models.NewTemplateAnpEpg("remove", path, Name, displayName, intraEpg, epgType, "", uSegEpg, intersiteMulticasteSource, preferredGroup, proxyArp, vrfRefMap, bdRefMap, cloudServiceEpgConfig)
```
Comment (akinross) [inline mso/resource_mso_schema_template_anp_epg.go:747]: Can you simplify the delete function by leverage the GetRemovePatchPayload function in the mso-go-client repo: https://github.com/ciscoecosystem/mso-go-client/blob/master/models/model.go#L40?

**Review by akinross — CHANGES_REQUESTED** [review]

File: mso/resource_mso_schema_template_anp_epg.go (hunk 1 above)
Comment (sajagana) [reply mso/resource_mso_schema_template_anp_epg.go:747]: Ok

File: mso/datasource_mso_schema_template_anp_epg.go [hunk 2]
```diff
@@ -21,87 +21,93 @@ func datasourceMSOSchemaTemplateAnpEpg() *schema.Resource {
...
+			"deployment_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"service_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"custom_service_type": {
+				Type:     schema.TypeString,
```
[the remark this answers is not in the document]
Comment (sajagana) [reply mso/datasource_mso_schema_template_anp_epg.go:110]: Yes

File: mso/datasource_mso_schema_template_bd.go [hunk 3]
```diff
@@ -15,151 +15,128 @@ func dataSourceMSOTemplateBD() *schema.Resource {
...
 						},
 						"dhcp_option_policy_version": &schema.Schema{
 							Type:     schema.TypeInt,
-							Optional: true,
 							Computed: true,
 						},
 					},
 				},
 			},
 			"dhcp_policies": &schema.Schema{
```
[the remark this answers is not in the document]
Comment (sajagana) [reply mso/datasource_mso_schema_template_bd.go:119]: Will track the TypeSet data source issue separately.

**Review by akinross — DISMISSED** [review]
LGTM

File: website/docs/r/schema_template_bd.html.markdown [hunk 4]
```diff
@@ -29,12 +29,13 @@ resource "mso_schema_template_bd" "bridge_domain" {
...
 * `display_name` - (Required) Display Name of the Bridge Domain on the MSO UI.
-* `vrf_name` - (Required) Name of VRF to attach with Bridge Domain. VRF must exist.
-* `vrf_schema_id` - (Optional) SchemaID of VRF. schema_id of Bridge Domain will be used if not provided. Should use this parameter when VRF is in different schema than BD.
-* `vrf_template_name` - (Optional) Template Name of VRF. template_name of Bridge Domain will be used if not provided. Should use this parameter when VRF is in different schema than BD.
-* `layer2_unknown_unicast` - (Optional) Type of layer 2 unknown unicast. Allowed values are `flood` and `proxy`. Default to `flood`.
+* `description` - (Optional) Description of the Bridge Domain on the MSO UI.
+* `vrf_name` - (Required) Name of the VRF to attach with Bridge Domain. VRF must exist.
+* `vrf_schema_id` - (Optional) SchemaID of the VRF. schema_id of Bridge Domain will be used if not provided. Should use this parameter when VRF is in different schema than BD.
+* `vrf_template_name` - (Optional) Template Name of the VRF. template_name of Bridge Domain will be used if not provided. Should use this parameter when VRF is in different schema than BD.
```
Comment (shrsr) [inline website/docs/r/schema_template_bd.html.markdown:37]: Template Name of the VRF. template_name of the Bridge Domain will be used if not provided. This parameter is required when the VRF is in a schema which is different from the schema of the BD.

**Review by shrsr — CHANGES_REQUESTED** [review]

**Review by shrsr — APPROVED** [review]
LGTM

**Review by anvitha-jain — APPROVED** [review]
LGTM

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**